server/README.md

```diff
@@ -95,20 +95,21 @@ note: recommended to use git bash inside vscode for all of this
 4. create a virtual environment in repo root (python -m venv venv)
 5. activate virtual environment (source venv/Scripts/activate or venv/bin/activate)
 6. install dependencies (`pip install -r requirements.txt -r requirements_dev.txt`)
-7. setting postgres:
+7. Start the db server: pg_ctl -D ~/pg/data/ -l logfile start
```

Comment:
Can we just add "(if applicable)" here? Only people who used the other guide for docker-less phac-windows will need to do this, but everyone will have to run the other commands

Thanks for updating the docs!